app/lib/pdf_filler/id39r_pdf.rb

```diff
@@ -16,9 +16,13 @@ def initialize(submission)
 
     def hash_for_pdf
       answers = {
-        "BL6" => @xml_document.at('Form39R ChildCareCreditAmt')&.text,
```

Comment:
i removed the "Form39R" to be more consistent here

Reply:
[dust] I think "Form39R" was a nice guard to have in place in case there's another `ChildCareCreditAmt` in the xml. Fortunately, the federal amount is called `FedChildCareCreditAmt`, so there's no chance of collision in this case, but it would have been reasonable to leave it in.

Reply:
Hmm would it be good practice to always prepend the selector with the form name?

Reply:
cc @powersurge360 since I asked why you did this, maybe this will be just good practice

Reply:
It's a good practice, but without a known naming collision I don't think changing it is needed.

Reply:
👍 just changed it since it was easy enough to do & I want people to follow this precedence just in case